pkg/virt-controller/services/vm.go

```diff
@@ -93,14 +93,14 @@ func (v *vmService) DeleteVMPod(vm *corev1.VM) error {
 	precond.MustNotBeNil(vm)
 	precond.MustNotBeEmpty(vm.GetObjectMeta().GetName())
```

Comment:
We were not really strict with the procond thing. It is meant to be a helper, to detect contract mismatches which should not be possible to happen, and to help you when writing tests, so that you have to pass in more realistic data ...

Would be nice if we would continue using it, or use it even more. We can discuss if we want to keep it.